### DISCUSSION TITLE
Add played> and played< search filters in song select

### DISCUSSION THREAD

**Discussion by Elvendir** [ppy/osu#22908] [Ideas] [answered]
In stable it is possible to filter beatmaps depending on last time played (using the key "played").
It is useful for looking at which maps where locally played during last day/month/year or hiding maps recently played.

**Comment by peppy** [comment, marked as answer]
We don't have any date criteria yet, but it shouldn't be a huge hurdle to add them.

**Reply by Elvendir** [reply]
Great thanks. Can I make a PR for that ?

**Reply by peppy** [reply]
Sure.
